src/components/CommitHistory/HistoryMain/History.css

```diff
@@ -48,10 +48,10 @@
 .SmallButton:hover {
   opacity: 0.7;
   transform: scale(1.03);
-  transition-duration: 0.5s;
```

Comment:
Maybe try implementing this transition for both the buttons, looks better that way.